unity-renderer/Assets/DCLPlugins/RealmsPlugin/Utils/JumpToHomeController.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using DCL;
+using DCL.Interface;
+using UnityEngine;
+using UnityEngine.UI;
+using Variables.RealmsInfo;
+
+public class JumpToHomeController : MonoBehaviour
+{
+
```

Comment:
I would remove spaces at the start and the end of the class. But you can keep it since we will reformat all project soon anyway (with the upcoming new .editorconfig)

Reply:
Niceee. I ll remove it manually, but cant wait for the .editorconfig  🤩